[2024-10-15T19:30:15.867+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-15T19:30:16.021+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: hello_world_dag.hello_world_task2 scheduled__2024-06-14T00:00:00+00:00 [queued]>
[2024-10-15T19:30:16.080+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: hello_world_dag.hello_world_task2 scheduled__2024-06-14T00:00:00+00:00 [queued]>
[2024-10-15T19:30:16.083+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-15T19:30:16.197+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): hello_world_task2> on 2024-06-14 00:00:00+00:00
[2024-10-15T19:30:16.229+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=2501) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-15T19:30:16.256+0000] {standard_task_runner.py:63} INFO - Started process 2533 to run task
[2024-10-15T19:30:16.332+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'hello_world_dag', 'hello_world_task2', 'scheduled__2024-06-14T00:00:00+00:00', '--job-id', '528', '--raw', '--subdir', 'DAGS_FOLDER/example_hello_world.py', '--cfg-path', '/tmp/tmp1u7243rh']
[2024-10-15T19:30:16.341+0000] {standard_task_runner.py:91} INFO - Job 528: Subtask hello_world_task2
[2024-10-15T19:30:16.557+0000] {task_command.py:426} INFO - Running <TaskInstance: hello_world_dag.hello_world_task2 scheduled__2024-06-14T00:00:00+00:00 [running]> on host f9b6cc3118f7
[2024-10-15T19:30:17.235+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='hello_world_dag' AIRFLOW_CTX_TASK_ID='hello_world_task2' AIRFLOW_CTX_EXECUTION_DATE='2024-06-14T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-14T00:00:00+00:00'
[2024-10-15T19:30:17.257+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-15T19:30:17.260+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-10-15T19:30:17.265+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', "echo 'Hello World! Now I am triggered after the first task!'"]
[2024-10-15T19:30:17.296+0000] {subprocess.py:86} INFO - Output:
[2024-10-15T19:30:17.301+0000] {subprocess.py:93} INFO - Hello World! Now I am triggered after the first task!
[2024-10-15T19:30:17.304+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-10-15T19:30:17.307+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-15T19:30:17.382+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=hello_world_dag, task_id=hello_world_task2, run_id=scheduled__2024-06-14T00:00:00+00:00, execution_date=20240614T000000, start_date=20241015T193016, end_date=20241015T193017
[2024-10-15T19:30:17.525+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-15T19:30:17.725+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-15T19:30:17.759+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
